images/nginx/rootfs/build.sh

```diff
@@ -413,14 +413,19 @@ git submodule update
 cd "$BUILD_PATH"
 git clone -b v3/master --single-branch https://github.com/SpiderLabs/ModSecurity
 cd ModSecurity/
-git checkout 973c1f1028429452308bcbce7df8a6283dc59ffe
+git checkout 18cdffdbca75e6b9f790f6df2807a32cd805c0a0
```

Comment:
@aledbf Can the e2e test image obtain the changes in this PR?

Reply:
@diazjf please remove this change ^^ and rebase from master to use the latest change introduced in #3353 and #3354